### PR TITLE
use tf without gpu support by default

### PR DESCRIPTION
**Pull request by wochinge**
**Proposed changes**:
- TensorFlow 1.15 for some reason includes the binaries for cpu and gpu which makes our docker images huge. Switching to the cpu only package.
- Rasa part of https://github.com/RasaHQ/rasa-x/issues/1643
- more information here https://www.tensorflow.org/install/pip

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: changelog/4989.improvement.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Switching back to a TensorFlow release which only includes CPU support to reduce the
+size of the dependencies. If you want to use the TensorFlow package with GPU support,
+please run ``tensorflow==1.15.0``.
```
Comment (dakshvar22) [inline changelog/4989.improvement.rst]:
```suggestion
please run ``tensorflow-gpu==1.15.0``.
```
Let's be specific about the gpu package as well?

**Comment by wochinge** [comment]
@dakshvar22 So `tensorflow` included only the cpu packages until 1.15. `tensorflow==1.15` included both gpu and cpu packages. For tensorflow 2 it's again the old pattern: `tensorflow==2.0` is only cpu, while `tensorflow-gpu==2.0` is the gpu package. So by using `tensorflow-cpu==1.15` we just go back to the way we have been releasing things before.

**Comment by wochinge** [comment]
@dakshvar22 Can I merge this?

**Comment by dakshvar22** [comment]
@wochinge Sounds good!

**Pull request merged**